### PR TITLE
ADEN-3097 Update evolve slot params in data provider

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

**Comment by drets** [comment]
LGTM :+1:

File: src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java [hunk 1]
```diff
@@ -48,19 +48,15 @@ public WebElement forElementPresent(By by) {
...
 
-      throw e;
-    } finally {
-      restoreDeaultImplicitWait();
-    }
+  /**
+   * Checks if the element is present in browser DOM
+   */
+  public WebElement forElementPresent(By by, boolean failOnTimeout, int timeout) {
+    WebDriverWait wait = new WebDriverWait(this.driver, timeout);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java]: MAJOR Rename "wait" which hides the field declared at line 32. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java [hunk 2]
```diff
@@ -416,4 +412,20 @@ public void forXMilliseconds(int time) {
       PageObjectLogging.log("Wait.forXMilliseconds", e, false);
     }
   }
+
+  private WebElement forElementPresent(WebDriverWait wait, By by, boolean failOnTimeout) {
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java]: Could you use https://github.com/Wikia/selenium-tests/blob/ADEN-3097-extend-evolve-params/src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java#L65 ?

**Comment by drets** [comment]
LGTM :+1:

**Pull request merged**